snmp/datadog_checks/snmp/data/conf.yaml.example

```
@@ -303,3 +303,15 @@ instances:
         name: tcpPassiveOpens
         metric_tags:
           - TCP
+
+    ## @param metric_tags - list of elements - optional
+    ## Specify tags that you want applied to all metrics. A tag can be applied from a symbol or an OID.
+    #
+    # metric_tags:
+    #  - # From a symbol
+    #    MIB: SNMPv2-MIB
+    #    symbol: sysName
+    #    tag: snmp_host
+    #  - # From an OID:
+    #    OID: 1.3.6.1.2.1.1.5
+    #    name: snmp_host
```

Looks like:

- `symbol` is required here too.
- Need to update `name -> tag` here too.

```suggestion
    #    symbol: sysName
    #    tag: snmp_host
```